Add README for KTX Sample (#143)
sendbird/sendbird-chat-sample-android: groupchannel-local-caching-ktx/README.md

# Sendbird Chat KTX sample for Android

## Overview

This project provides KTX sample for SendbirdChat KTX SDK.

For more information on SendbirdChat KTX, refer to [Kotlin extension (KTX)](https://sendbird.com/docs/chat/sdk/v4/android/getting-started/kotlin-extension).

## Running the app

> Open this project in Android Studio and select 'run configurations' to view a dropdown of feature-level Chat samples to choose from.

Reviewer: ```suggestion
Open this project in Android Studio and select 'run configurations' and choose `groupchannel-local-caching-ktx` from a dropdown.
```
